keyboards/dz60/keymaps/jkbone/keymap.c

```diff
@@ -0,0 +1,70 @@
+#include "dz60.h"
```

Comment:
Could you replace this includes with `#include QMK_KEYBOARD_H`

Reply:
done!